.eslintignore

```diff
@@ -1,5 +1,12 @@
-.circleci/*
-.githooks/*
-.idea/*
-.vscode/*
-node_modules/*
+.circleci
+.githooks
+.idea
+.vscode
+.yarn*
```

Comment:
What is this meant to match against?

Reply:
Prevents parsing of any yarn files and esp. the .yarn directory though passing an `--ext .ts` option to the `eslint` command is arguably more efficient.